src/config/validation.ts

```diff
@@ -0,0 +1,15 @@
+import AJV = require('ajv');
```

Comment:
Another “config” layover. This PR should not have the term “config” in it _anywhere_. 😅

Reply:
Like I said: I think setting custom rulesets _is configuration_. What do you want me to change? folder name? To what? :) `rulesets` is already taken ;)

Reply:
Already taken? That just means you can put the files inside without having to make a new directory.